context/app/static/js/components/search/Results/ResultsTable.tsx

```diff
@@ -103,6 +103,12 @@ function CellContent({ field, fieldValue }: { field: string; fieldValue: string
     case 'hubmap_id':
       return <InternalLink href={`/browse/${fieldValue}`}>{fieldValue}</InternalLink>;
     case 'last_modified_timestamp':
+    case 'published_timestamp':
+    case 'created_timestamp':
+      // Handle datasets without published timestamps.
+      if (!fieldValue) {
+        return null;
```

Comment:
Could we display a placeholder of some sort in this case to convey to users that there is supposed to be nothing there/that it's not broken? An em-dash (`—`) could work well for these cases, or `N/A`.

Reply:
I'll follow up with Tiffany and make a sweeping change for all cells. For now, this defaults to the behavior for other cells.